charts/nextcloud/templates/notify_push/deployment.yaml

```diff
@@ -0,0 +1,73 @@
+{{- if .Values.notifyPush.enabled }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "nextcloud.fullname" . }}-notify-push
+  labels:
+    app.kubernetes.io/name: {{ include "nextcloud.name" . }}
+    helm.sh/chart: {{ include "nextcloud.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/component: notify-push
+spec:
+  replicas: {{ .Values.notifyPush.replicaCount }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "nextcloud.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+      app.kubernetes.io/component: notify-push
+  template:
+    metadata:
+      annotations:
+        {{- toYaml .Values.notifyPush.podAnnotations | nindent 8 }}
+      labels:
+        app.kubernetes.io/name: {{ include "nextcloud.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        app.kubernetes.io/component: notify-push
+        {{- with .Values.notifyPush.podLabels }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+      {{- with .Values.notifyPush.image.pullSecrets }}
+      imagePullSecrets:
+        {{- range . }}
+        - name: {{ . }}
+        {{- end}}
+      {{- end }}
+      containers:
+        - name: notify-push
+          {{- with .Values.notifyPush.image }}
+          image: "{{ .registry }}/{{ .repository }}:{{ .tag }}"
+          imagePullPolicy: {{ .pullPolicy }}
+          {{- end }}
+          env:
+            - name: PORT
+              value: "7867"
+            - name: METRICS_PORT
+              value: "9867"
+            {{- include "nextcloud.env.database" . | nindent 12 }}
+            {{- with .Values.notifyPush.redisURLEnv }}
+            - name: "REDIS_URL"
+              {{- toYaml . | nindent 14 }}
+            {{- else }}
+            {{- if not .Values.redis.enabled }}
+            {{- fail "notify-push needs an redis (redis.enabled=true or notifyPush.redisURLEnv={...})" }}
```

Comment:
Good to have it here, but it would be nice to mention this in the readme as well.